docs/assembly.rst

```diff
@@ -861,38 +861,37 @@ Grammar::
     AssemblyItem =
         Identifier |
         AssemblyBlock |
-        FunctionalAssemblyExpression |
+        AssemblyExpression |
         AssemblyLocalDefinition |
-        FunctionalAssemblyAssignment |
         AssemblyAssignment |
+        AssemblyStackAssignment |
         LabelDefinition |
         AssemblyIf |
         AssemblySwitch |
         AssemblyFunctionDefinition |
         AssemblyFor |
-        'break' | 'continue' |
-        SubAssembly | 'dataSize' '(' Identifier ')' |
```

Comment:
These are actually used in the output generated with `--asm`.

Reply:
The only one I removed of those is the dataSize rule, which seemed to add no more expressiveness grammar-wise than the assemblyCall.

Reply:
It is supposed to be a special keyword, but well. I'll merge.